views.go

```diff
@@ -542,6 +537,9 @@ func (lk *logKeeper) viewTestByBuildIdTestId(w http.ResponseWriter, r *http.Requ
 		if emptyLog {
 			lk.render.WriteJSON(w, http.StatusOK, nil)
 		}
+	} else if len(r.FormValue("html")) == 0 {
```

Comment:
Are we ever using this form value anywhere? Maybe we should drop the old UI? Otherwise lgtm.